includes/class.llms.post-types.php

```diff
@@ -477,13 +653,14 @@ public static function register_post_types() {
 				'exclude_from_search' => false,
 				'hierarchical'        => false,
 				'rewrite'             => array(
-					'slug'       => _x( 'course', 'course url slug', 'lifterlms' ),
+					'slug'       => $rewrite_slugs['post_types']['course']['slug'] ?? '',
```

Comment:
Do we want to have these default to `_x( ... )` instead of an empty slug if the option value doesn't exist for some reason? If it's saved it should be a non-issue, though.